packages/@eventual/aws-cdk/src/entity-service.ts

```diff
@@ -368,6 +373,7 @@ export class EntityStream extends Construct implements EventualResource {
 
     props.serviceProps.bucketService.configureReadWriteBuckets(this.handler);
     props.entityService.configureReadWriteEntityTable(this.handler);
+    props.serviceProps.searchService?.configureSearch(this.handler);
```

Comment:
I agree that a `service.configureWorkerIntrinsics` would be nice to match the `registerWorkerIntrinsics` method.

Reply:
Totally. It's a mess